CMakeLists.txt

```diff
@@ -291,7 +291,9 @@ add_custom_target(pack-skins ALL
                   DEPENDS TexturePacker::TexturePacker export-files ${XBT_FILES})
 set_target_properties(pack-skins PROPERTIES FOLDER "Build Utilities")
 
-file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/system/players/VideoPlayer)
```

Comment:
this has to stay as is, system/players/VideoPlayer/ is used for other stuff like libdvdnav